.github/workflows/ubuntu_install.yml

```diff
@@ -38,8 +34,10 @@ jobs:
       - name: Install
         run: cmake --install build
       - name: Prepare test package
-        run: cmake -DCMAKE_INSTALL_PREFIX:PATH=../../destination -S tests/installation -B buildbabyada
+        run: cmake -DCMAKE_INSTALL_PREFIX:PATH=../../destination -S tests/installation |
```

Comment:
We need to either pass CC=gcc-12 CXX=g++-12 or do `sudo ln -sf /usr/bin/gcc-12 /usr/bin/gcc` kind of symlink to fix the issue. Ubuntu 22 doesn't include gcc 12 or something newer

Reply:
Let us upgrade to ubuntu 24.

Messing around with paths is fragile.